runtimes/eoapi/vector/eoapi/vector/app.py

```diff
@@ -16,7 +17,8 @@
 from tipg.settings import PostgresSettings
 
 from . import __version__ as eoapi_vector_version
-from . import auth, config, logs
+from .config import ApiSettings
+from .logs import init_logging
```

Comment:
personal opinion, I tend to avoid importing full submodule if it's only for one variable/class/function